src/components/data/banking/LendingRate.js

```diff
@@ -61,7 +61,7 @@ const LendingRate = (props) => {
       {!!applicationFrequency && <div>Applied <Duration prefix="every" value={applicationFrequency}/></div>}
       {!!interestPaymentDue && <div>Interest Payment {translateInterestPaymentDue(interestPaymentDue)}</div>}
       {!!repaymentType && <div>Repayment Type {translateRepaymentType(repaymentType)}</div>}
-      {!!loanPurpose && <div>Loan Purpose {loanPurpose}</div>}
+      {!!loanPurpose && <div>Loan Purpose {translateloanPurpose (loanPurpose)}</div>}
```

Comment:
Sorry for neat-picking, but could you please remove the space between  translateloanPurpose and the opening bracket?

Reply:
updated